docs/federation.md

```diff
@@ -0,0 +1,126 @@
+## Federation
+
+AMKO has a federator which can federate the AMKO configuration to a set of member clusters. This is done to ensure that a user can initiate a recovery whenever there's some kind of a disaster (without the need of backing up the AMKO configs).
```

Comment:
AMKO uses federation to replicate AMKO configuration to a set of member clusters. This ensures a seamless recovery of AMKO configs for disaster recovery.